tools/lkl/tests/net.sh

```diff
@@ -43,15 +43,19 @@ fi
 if ! [ -z $DST ]; then
     echo "== RAW socket (LKL net) tests =="
     sudo ip link set dev ${IFNAME} promisc on
-    sudo ./net-test raw ${IFNAME} ${DST} dhcp
+    sudo ./net-test raw ${IFNAME} ${DST} dhcp &
+    sleep 5
+    sudo killall -q net-test && (echo "killed, skipped")
```

Comment:
if kilall doesn't find, it return 0, keep quiet (-q) and goto next
if killall finds, return 1 and report via echo ..., but no error.

hmm, there is no chance to report an error with dhcp test now..
if there were a way to predict that dhcp works, this check can be expanded but dunno how.